ivy/functional/backends/jax/data_type.py

```diff
@@ -131,12 +131,16 @@ def broadcast_to(
 
 
 @_handle_nestable_dtype_info
-def finfo(type: Union[jnp.dtype, str, JaxArray], /) -> Finfo:
+def finfo(type: Union[jnp.dtype, str, JaxArray, np.ndarray], /) -> Finfo:
+    if isinstance(type, np.ndarray):
+        type = type.dtype.name
     return Finfo(jnp.finfo(ivy.as_native_dtype(type)))
```

Comment:
This function is not meant to accept ndarrays, just dtype objects

Reply:
Actually from the array API they specified that they can accept array as well
https://data-apis.org/array-api/2022.12/API_specification/generated/array_api.iinfo.html